### PR TITLE
Check site in fedcloudclient

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

Following up the discussion in https://github.com/tdviet/fedcloudclient/pull/169 we wanted to have a more automated way of checking consistencies between this repository and the list hosted on the `fedcloudclient` repository.

This is a first attempt to address it.

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
Nice! Can you check the linter errors:
```2023-01-12 08:48:46 [ERROR]   Error code: 1. Command output:
------

In /github/workspace/validate.sh line 65:
egrep --invert-match --regexp='#' --regexp='^$' "$FEDCLOUD_CLI_SITES" \
^---^ SC21[96](https://github.com/EGI-Federation/fedcloud-catchall-operations/actions/runs/3900595242/jobs/6661435374#step:4:97) (info): egrep is non-standard and deprecated. Use grep -E instead.


In /github/workspace/validate.sh line 70:
for site in $(cat "$SITES_CHECK")
            ^-------------------^ SC2013 (info): To read lines rather than words, pipe/redirect to a 'while read' loop.


In /github/workspace/validate.sh line 72:
    if ! [ -s $site ]
              ^---^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: 
    if ! [ -s "$site" ]

For more information:
  https://www.shellcheck.net/wiki/SC2013 -- To read lines rather than words, ...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
  https://www.shellcheck.net/wiki/SC2196 -- egrep is non-standard and depreca...
```

**Comment by sebastian-luna-valero** [comment]
oh, sorry!

for some reason I missed those errors!

File: validate.sh [hunk 1]
```diff
@@ -6,9 +6,14 @@ exit_value=0
 
 # Get all VOs names
 VO_LIST=$(mktemp)
-curl -s "http://cclavoisier01.in2p3.fr:8080/lavoisier/VoList?accept=json" \
+curl --silent "http://cclavoisier01.in2p3.fr:8080/lavoisier/VoList?accept=json" \
     | jq -r ".data[].name" > "$VO_LIST"
 
+# Get fedcloudclient sites
+FEDCLOUD_CLI_SITES=$(mktemp)
+curl --output "$FEDCLOUD_CLI_SITES" \
```
Comment (enolfc) [inline validate.sh:14]: Maybe we should harmonise how curl is used

**Review by enolfc — CHANGES_REQUESTED** [review]
Could you either change the first or the second curl so they look similar?

**Comment by sebastian-luna-valero** [comment]
Agreed!

**Review by enolfc — APPROVED** [review]
LGTM!

**Pull request merged**